classifides/accounts/views.py

```diff
@@ -0,0 +1,41 @@
+from django.shortcuts import render, redirect
+from django.contrib.auth import login, authenticate
+from django.contrib.auth.models import User
+from django.contrib.auth.decorators import login_required
+from ads.models import Ad  
+from .forms import CustomUserCreationForm
+from django.contrib.auth import logout
+
+def register(request):
+    if request.method == 'POST':
+        form = CustomUserCreationForm(request.POST)
+        if form.is_valid():
+            email = form.cleaned_data.get('email')
+            if User.objects.filter(email=email).exists():
+                form.add_error('email', 'Email is already in use')
```

Comment:
This validation should be done inside the form.

Reply:
ok